### PR TITLE
Correct page not found links to docs

### PR DESCRIPTION
**Pull request by BabylonJSGuide**
Corrected some links that did not point to any page within the docs. Probably they were missing from redirects. Now point directly to doc page.

### CODE REVIEW

**Review by deltakosh — APPROVED** [review]

File: src/Engines/Extensions/engine.views.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Scene } from '../../scene';
 
 /**
  * Class used to define an additional view for the engine
- * @see https://doc.babylonjs.com/how_to/multi_canvases
+ * @see https:///multi_canvases
```
Comment (sebavan) [inline src/Engines/Extensions/engine.views.ts:8]: Sounds like a typo

File: src/Materials/Textures/dynamicTexture.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { ICanvas, ICanvasRenderingContext } from "../../Engines/ICanvas";
 
 /**
  * A class extending Texture allowing drawing on a texture
- * @see https://doc.babylonjs.com/how_to/dynamictexture
+ * @see https:///dynamictexture
```
Comment (sebavan) [inline src/Materials/Textures/dynamicTexture.ts:12]: Sounds like a typo

File: src/Materials/Textures/videoTexture.ts [hunk 3]
```diff
@@ -41,7 +41,7 @@ export interface VideoTextureSettings {
 /**
  * If you want to display a video in your scene, this is the special texture for that.
  * This special texture works similar to other textures, with the exception of a few parameters.
- * @see https://doc.babylonjs.com/how_to/video_texture
+ * @see https:///video_texture
```
Comment (sebavan) [inline src/Materials/Textures/videoTexture.ts:44]: Sounds like a typo

File: src/Materials/fresnelParameters.ts [hunk 4]
```diff
@@ -67,7 +67,7 @@ export type IFresnelParametersSerialized = {
 
 /**
  * This represents all the required information to add a fresnel effect on a material:
- * @see https://doc.babylonjs.com/how_to/how_to_use_fresnelparameters
+ * @see https:///how_to_use_fresnelparameters
```
Comment (sebavan) [inline src/Materials/fresnelParameters.ts:70]: Sounds like a typo

File: src/Materials/shaderMaterial.ts [hunk 5]
```diff
@@ -95,7 +95,7 @@ export interface IShaderMaterialOptions {
  *
  * This returned material effects how the mesh will look based on the code in the shaders.
  *
- * @see https://doc.babylonjs.com/how_to/shader_material
+ * @see https:///shader_material
```
Comment (sebavan) [inline src/Materials/shaderMaterial.ts:98]: Sounds like a typo

**Review by sebavan — CHANGES_REQUESTED** [review]
Looks like a few links are not valid @BabylonJSGuide ?

**Comment by BabylonJSGuide** [comment]
@sebavan -sorry about that, some sort of accidental delete from files. Have corrected.

File: src/Engines/Extensions/engine.views.ts (hunk 1 above)
Comment (BabylonJSGuide) [reply src/Engines/Extensions/engine.views.ts]: Done

File: src/Materials/Textures/dynamicTexture.ts (hunk 2 above)
Comment (BabylonJSGuide) [reply src/Materials/Textures/dynamicTexture.ts]: Done

File: src/Materials/Textures/videoTexture.ts (hunk 3 above)
Comment (BabylonJSGuide) [reply src/Materials/Textures/videoTexture.ts]: Done

File: src/Materials/fresnelParameters.ts (hunk 4 above)
Comment (BabylonJSGuide) [reply src/Materials/fresnelParameters.ts]: Done

File: src/Materials/shaderMaterial.ts (hunk 5 above)
Comment (BabylonJSGuide) [reply src/Materials/shaderMaterial.ts]: Done

**Comment by sebavan** [comment]
do not be sorry, this is exactly why we have reviews :-) Thanks a lot for the contrib

**Pull request merged**